src/main/resources/openapi/paths/example/example.yaml

```diff
@@ -0,0 +1,34 @@
+get:
+  tags:
+    - Example
+  summary: Get example object
+  operationId: get-examples
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '../../models/example/example-data-response.yaml#/ExampleDataResponse'
```

Comment:
Da się to zrobić bez referencji do plików tak, żeby to było tylko w openapi.yaml? 

Bo wtedy łatwiej byłoby używać openapi i nie musisz X plików podawać np. dla frontu